clients/cloud/scala/src/main/resources/ccloud.properties

```diff
@@ -0,0 +1,7 @@
+ssl.endpoint.identification.algorithm=https
+sasl.mechanism=PLAIN
+request.timeout.ms=20000
```

Comment:
unless request.timeout.ms and retry.backoff.ms are needed for this example, we should leave them out to keep things light and airy